backend/models/eventTemplate.model.js

```diff
@@ -1,20 +1,27 @@
 const mongoose = require('mongoose');
-
+const recurIntervalEnums = ['daily','weekly','bi-weekly','monthly'];  // daily, weekly, bi-weekly, monthly 
```

Comment:
I'm not sure that making variables for all of these really buys you anything over than doing it inline.  It seems nice and tidy, but it's also an extra line of code that you have to search for if you are debugging, or want to make changes.  It's a style thing, and I don't really have a problem with it, but if you are going to make variables for these, then I'd recommend making it consistent and doing a variable for 'type' as well.